### PR TITLE
BETA: Register devendra.is-a.dev

### PR DESCRIPTION
**Pull request by devendrakanojiya**
Added `devendra.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by devendrakanojiya** [comment]
@CuteDog5695 bro check .

File: domains/devendra.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devendrakanojiya",
+        "email": "s_36100@bkbirlacollegekalyan.com"
+    },
+    "record": {
+        "CNAME": "devendrakanojiya.github.io"
```
Comment (CuteDog5695) [inline domains/devendra.json:7]: page gives 404 page

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/devendra.json (hunk 1 above)
Comment (devendrakanojiya) [reply domains/devendra.json:7]: my github page is running on  https://devendrakanojiya.github.io/me

but as far I know we don't have to add "/me" in record  or we have to ?

Comment (devendrakanojiya) [reply domains/devendra.json:7]: fixed it 👍

**Review by iostpa — APPROVED** [review]
now its fine

**Pull request merged**